lib/utils/twake_snackbar.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/utils/responsive/responsive_utils.dart';
+import 'package:fluffychat/widgets/twake_app.dart';
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+
+class TwakeSnackBarStyle {
+  static ResponsiveUtils responsiveUtils = getIt.get<ResponsiveUtils>();
+
+  static const EdgeInsetsDirectional snackBarPadding =
+      EdgeInsetsDirectional.symmetric(
+    horizontal: 16,
+    vertical: 14,
+  );
+
+  static double? widthSnackBar(BuildContext context) {
+    if (responsiveUtils.isWebDesktop(context)) {
+      return 334;
+    } else {
+      return null;
+    }
+  }
+}
+
+class TwakeSnackBar {
+  static void show(String message) {
+    final context = TwakeApp.routerKey.currentContext;
+    if (context == null) {
+      Fluttertoast.showToast(msg: message);
+      return;
+    }
```

Comment:
so it still can show the old toast, how about using context as an optional parameter ? and use scaffold only

Reply:
the idea behind this helper is that it can be used everywhere (widget or not widget) without explicitly specifying the build context.

IMO it's good like this, I used Fluttertoast to be safe if context is null